src/olympia/devhub/views.py

```diff
@@ -1490,32 +1490,33 @@ def submit_finish(request, addon_id, addon):
     # Bounce to the versions page if they don't have any versions.
     if not addon.versions.exists():
         return redirect(addon.get_dev_url('versions'))
-    sp = addon.current_version.supported_platforms
-    is_platform_specific = sp != [amo.PLATFORM_ALL]
+    uploaded_version = addon.versions.latest()
```

Comment:
The issue is, with the way things are set up atm we don't know the channel of the version we just finished uploading, so I can't give a channel to `find_latest_version()`. I could use that now, but I'd have to change it back if/when we force `find_latest_version()` to always pass a channel.

Maybe that's safer for now, though.